src/graphql/queries/leaderboard.js

```diff
@@ -23,13 +24,20 @@ const eventLeaderboardEntries = async (
   { db: { EventLeaderboard } },
 ) => {
   const query = filter ? { $or: buildFilters(lowercaseFilters(filter)) } : {};
-  return runPaginatedQuery({
+  const res = await runPaginatedQuery({
     db: EventLeaderboard,
     filter: query,
     orderBy,
     limit,
     skip,
   });
+
+  if (orderBy && orderBy.length > 0) {
+    const { field } = orderBy[0];
+    res.items.sort((a, b) => new BigNumber(b[field]).comparedTo(new BigNumber(a[field]))); // all descending
```

Comment:
what if trying to sort by address?

Reply:
nit: suggest to maybe use `BN.js` in this case. not sure if the performance impacts with `BigNumber` would come into play here.

Reply:
`field` here is either `investments` or `winnings`, which are all converted from `BigNumber` instances, think it's better to keep it consistent to use `BigNumber`?

Reply:
1. Need to handle what happens if  someone passes in `{ field: 'eventAddress', direction: 'ASC' }` or `betterAddress`. It will crash here since you're trying to convert an address to a BigNumber.
2. Was just worried about the performance of it. If you feel the performance is good compared to BN, then let's just keep it BigNumber. But would like to still use BN whenever possible to keep it standardized throughout the code.

Reply:
ratio is float type, think it's better keep it consistent to use Bignumber

I will skip the sorting if it's address

Reply:
Ah right, ya if dealing w/ floats, then BigNumber all the way.